### ISSUE TITLE
More shortcuts

### ISSUE THREAD

**Issue by Shywizz** [Keksuccino/FancyMenu#830]
Using the mouse is obligatory for something like this, but I have noticed a lack in "basic" keyboard navigation for context menus, sub-menus widget screens (text, action script, image/asset browser)

It would be really nice if, instead of always having to rely on the mouse I could, for example : 
- Navigate right click context menus for widgets with the keyboard
- Use the [Menu Key](https://en.wikipedia.org/wiki/Menu_key) to trigger right clicks where possible
- Support basic Ctrl+(A|X|C|V) Editing for textboxes
- Tab "cyclcing" or arrow key navigation for screens that could support it ?
- Keyboard selection (seems broken so far, only works with the mouse)

Overhaul, the above suggested improvements would help streamline the layout creation workflow

**Comment by Keksuccino** [comment]
> Support basic Ctrl+(A|X|C|V) Editing for textboxes
What does A and X do? Text boxes and the editor both already support C and V if I remember correctly.

> Tab "cyclcing" or arrow key navigation for screens that could support it ?
FancyMenu's screens don't work like that, so no, that will not happen, sorry.

> Keyboard selection (seems broken so far, only works with the mouse)
Same as answer above.

**Comment by Keksuccino** [comment]
Decided not to add these, sorry.
Some of them are already part of FM and the others are not possible or very difficult to implement.
